packages/frontend/src/ee/features/aiCopilot/components/ChatElements/ToolCalls/AiProposeChangeToolCall/OperationRenderer.tsx

```diff
@@ -0,0 +1,42 @@
+import { assertUnreachable, capitalize } from '@lightdash/common';
+import { Stack, Text } from '@mantine-8/core';
+import { Paper } from '@mantine/core';
+import type { Operation } from './types';
+
+type ReplaceOperationProps = {
+    value: string;
+    fieldName: string;
+};
+
+const ReplaceOperation = ({ value, fieldName }: ReplaceOperationProps) => {
+    return (
+        <Paper bg="gray.0" p="xs" component={Stack} gap="xxs">
+            <Text component="code" size="xs" fw={600} c="gray.7">
+                {capitalize(fieldName)}
+            </Text>
+            <Text size="sm">"{value}"</Text>
+        </Paper>
+    );
+};
+
+type OperationRendererProps = {
+    operation: Operation;
+    fieldName: string;
```

Comment:
from offline convo: call "property" instead of "field"/"fieldName" to avoid mixing other parts in the codebase mentioning field, which is usually a queryable object(dim,metric,tablecalc...)